Themes.md

```diff
@@ -0,0 +1,130 @@
+# Themes (WIP), Changeable Styles
```

Comment:
I wrote a first version instead of the WIP (Write custom themes should be improved, but I think we can do it as soon we have the first feedbacks from the other users)